### PR TITLE
402 playwright for chart testing

### PR DESCRIPTION
**Pull request by Ganthry**

### CODE REVIEW

**Comment by Ganthry** [comment]
I've got playwright setup. Everything except for webkit is being cached, as theres some really strange edge case going on with the intersection of xvfb, webkit, and caching that makes things inconsistent. 

When running tests locally, there’s no guarantee that all of them will pass., (I've noticed I fail more tests based on how strong my network is or how charged my computer is 👎. Running more tests alone helps the tests be more consistent when running locally. Luckily they're consistent in CI.

If any playwright tests do fail in e2e, the pipeline comments all the playwright traces which can be viewed to help debug each layer of playwright test to see what went wrong, or specific tests can be run alone locally.

image
image

I go through some general tests, going to tabs, logging in, generating charts, downloading charts, changing charts with settings, loading custom settings, map points etc... 

The total pipeline takes about 20 minutes. If this is too long I can get rid of, or merge some of these tests. Also if there are any areas that should be covered, that aren't, I can add the tests in.

File: package.json [hunk 1]
```diff
@@ -17,6 +18,8 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-check-file": "^2.7.1",
     "eslint-plugin-react": "^7.33.2",
+    "form-auto-content": "^3.2.1",
+    "playwright": "^1.51.1",
```
Comment (paologum) [inline package.json:22]: we need form-auto-content on the root packages?

File: server/public/aboutPictures/EllisSelznick.jpg
[the remark this answers is not in the document]
Comment (Ganthry) [reply server/public/aboutPictures/EllisSelznick.jpg:1]: you know a lot of people have been saying this

**Review by paologum — APPROVED** [review]
